### ISSUE TITLE
arp-scan no results displayed 

### ISSUE THREAD

**Issue by sonikbhoom** [royhills/arp-scan#46]
```
09:56:33 [root@localhost ~]# arp-scan --interface=enp2s0 --localnet 
Interface: enp2s0, datalink type: EN10MB (Ethernet)
Starting arp-scan 1.9.5 with 256 hosts (https://github.com/royhills/arp-scan)

18 packets received by filter, 0 packets dropped by kernel
Ending arp-scan 1.9.5: 256 hosts scanned in 1.936 seconds (132.23 hosts/sec). 0 responded```


note that packets are received but not displayed.

This is occurring on three separate machines.  Fedora 30 & 31 and VM with kali-rolling.
Any ideas?

**Comment by royhills** [comment]
Fixed in arp-scan 1.9.6 - see #42 

The person responsible for producing the packages for the distro you're using should packages up the latest version or arp-scan.  Or you can manually install from the latest git release.

**Comment by sonikbhoom** [comment]
confirmed.  ```yum downgrade libpcap-1.9.0-3.fc30.x86_64 libpcap-1.9.0-3.fc30.i686``` fixes output
